README.md

```diff
@@ -20,6 +20,12 @@ node -r esbuild-register file.ts
 
 It will use `jsxFactory`, `jsxFragmentFactory` and `target` options from your `tsconfig.json`
 
+You can also pass a JSON string to specify configuration options for esbuild, such as if I want to pass in the banner configuration.
+
+```bash
+node -r esbuild-register file.ts --esbuild-config="{\"banner\": \"console.log('is banner')\"}"
+```
```

Comment:
I like this idea, but personally would prefer an option to specify a config-file that exports an object matching the esbuild config options.

Reply:
Ok, I'll resubmit when I have time, thanks for the reply

Reply:
I'm not the package author @seho-dev, so maybe wait until a reply if they agree with my person opinion :)